cli/proto/development.proto

```diff
@@ -0,0 +1,29 @@
+// FIXME: this is a copy of /development_server/proto/development.proto
```

Comment:
Do we have to do anything here? Why did we have to copy this fella?

Reply:
It's used in `./cli/main.rs:18`:

```rs
pub mod development {
    tonic::include_proto!("development");
}
```

Attempting to share .proto files across workspaces

Reply:
Next step would be to also move the entirety of tonic into its own package I assume?